website/src/tabs/rest/GetProgram.js

```diff
@@ -5,45 +5,77 @@ import { CopyButton } from "../../components/CopyButton";
 
 export const GetProgram = () => {
     const [program, setProgram] = useState(null);
-    const [programName, setProgramName] = useState(null);
+    const [programID, setProgramID] = useState(null);
+    const [status, setStatus] = useState("");
```

Comment:
Each of the states in these `useState` vars modify the same var if they're named the same way. These are all named status. Is this by design? I wonder if there's a way state could get out of order. Seems really low risk honestly though, and I'm not %100 sure what the best state management practice is in a lightweight web app like this so this is probably fine.

Just something to note.

Reply:
I'm not sure what the best React practice is here. When clicking on the webpage I do not not notice any strange state sharing between `status` variables. I will do some research and add any idiomatic changes in a follow up PR.